po/README.md

```diff
@@ -0,0 +1,3 @@
+### Translations
+You can translate Open Food Facts at https://translate.openfoodfacts.org. Please do not translate directly in the code, or your translations will be overwritten.
+If you have added or modified a string, you should update the crowdin-trigger branch from main/develop. This will trigger import within Crowdin, and 1 hour later, it will create a new pull request with added or modified strings in all language files, plus all added or modified translations.
```

Comment:
```suggestion
If you have added or modified a string in the source file (the file with the .pot extension), you should update the crowdin-trigger branch from main/develop. This will trigger import within Crowdin, and 1 hour later, it will create a new pull request with added or modified strings in all language files, plus all added or modified translations.
```